ibis/backends/conftest.py

```diff
@@ -25,6 +25,8 @@
 if TYPE_CHECKING:
     from collections.abc import Iterable
 
+    from ibis.backends.datafusion.tests.conftest import BackendTest
```

Comment:
I think you probably want a non-backend-specific class here:

```suggestion
    from ibis.backends.tests.base import BackendTest
```